### PR TITLE
yazi: new, 0.1.5

### PR DESCRIPTION
**Pull request by eatradish**
Topic Description
-----------------

yazi: new, 0.1.5

Package(s) Affected
-------------------

yazi: new, 0.1.5

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Comment by eatradish** [comment]
`riscv64` does not support `LuaJIT` :

```
  lj_arch.h:73:2: error: #error "No support for this architecture (yet)"
     73 | #error "No support for this architecture (yet)"
```

**Comment by MingcongBai** [comment]
> `riscv64` does not support `LuaJIT` :
> 
> ```
>   lj_arch.h:73:2: error: #error "No support for this architecture (yet)"
>      73 | #error "No support for this architecture (yet)"
> ```

Just skip it for now.

File: app-utils/yazi/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,10 @@
+PKGNAME="yazi"
+PKGDES="An terminal file manager"
```
Comment (MingcongBai) [inline app-utils/yazi/autobuild/defines]: A file manager for terminals

File: app-utils/yazi/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+PKGDEP="glibc gcc-runtime"
+BUILDDEP="rustc"
+PKGSEC=utils
+
+ABSPLITDBG=0
+
+# mlua-sys must be compiled with gcc
```
Comment (MingcongBai) [inline app-utils/yazi/autobuild/defines]: Mark this as `# FIXME:`

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: app-utils/yazi/autobuild/defines [hunk 3]
```diff
@@ -0,0 +1,10 @@
+PKGNAME="yazi"
+PKGDES="A file manager for terminal"
```
Comment (MingcongBai) [inline app-utils/yazi/autobuild/defines]: for terminals

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**